embodiedscan/structures/bbox_3d/utils.py

```diff
@@ -218,25 +219,24 @@ def get_box_type(box_type: str) -> Tuple[type, int]:
     Returns:
         tuple: Box type and box mode.
     """
-    from .box_3d_mode import (Box3DMode, CameraInstance3DBoxes,
-                              DepthInstance3DBoxes, EulerCameraInstance3DBoxes,
-                              EulerDepthInstance3DBoxes, LiDARInstance3DBoxes)
+    from .box_3d_mode import Box3DMode
+    from .euler_depth_box3d import EulerDepthInstance3DBoxes
     box_type_lower = box_type.lower()
-    if box_type_lower == 'lidar':
-        box_type_3d = LiDARInstance3DBoxes
-        box_mode_3d = Box3DMode.LIDAR
-    elif box_type_lower == 'camera':
-        box_type_3d = CameraInstance3DBoxes
-        box_mode_3d = Box3DMode.CAM
-    elif box_type_lower == 'depth':
-        box_type_3d = DepthInstance3DBoxes
-        box_mode_3d = Box3DMode.DEPTH
-    elif box_type_lower == 'euler-depth':
+    # if box_type_lower == 'lidar':
+    #     box_type_3d = LiDARInstance3DBoxes
+    #     box_mode_3d = Box3DMode.LIDAR
+    # elif box_type_lower == 'camera':
+    #     box_type_3d = CameraInstance3DBoxes
+    #     box_mode_3d = Box3DMode.CAM
+    # elif box_type_lower == 'depth':
+    #     box_type_3d = DepthInstance3DBoxes
+    #     box_mode_3d = Box3DMode.DEPTH
+    if box_type_lower == 'euler-depth':
```

Comment:
Add a comment: We temporarily only support EulerDepthInstance3DBoxes to support 9-DoF box operations and will consider refactoring this class with further experience.